README.md

```diff
@@ -1,13 +1,15 @@
 # commerce-sdk-isomorphic
 
-The Salesforce Commerce SDK (Isomorphic) allows easy interaction with the B2C Commerce platform’s Shopper APIs on the Node.js runtime and works both in browsers and Node applications. For a Node-based SDK that can access the Admin APIs in addition to the Shopper APIs, see the main [Commerce SDK](https://github.com/SalesforceCommerceCloud/commerce-sdk). 
+This SDK provides a Browser & Node.js JavaScript client for calling [B2C Commerce Shopper APIs](https://developer.salesforce.com/docs/commerce/commerce-api/overview).
+
+_For a Node.js only SDK that can also access Admin APIs checkout [Commerce SDK](https://github.com/SalesforceCommerceCloud/commerce-sdk)._
 
 ## Getting Started
 
 ### Requirements
 
 - Node `^12.x`, `^14.x`, `^16.x`, `^18.x` or `^20.x`
-
+- The SDK requires B2C Commerce API (SCAPI) to be configured. For more info see [Getting started with SCAPI](https://developer.salesforce.com/docs/commerce/commerce-api/guide/get-started.html).
```

Comment:
The pre-reqs for using SCAPI are described well here. We don't need to repeat this info in the README.